README.md

```diff
@@ -34,6 +34,19 @@ In the repository, you can find:
   suite](https://github.com/c4dt/lightarti-rest-android/blob/main/artiwrapper/src/androidTest/java/org/c4dt/artiwrapper/JniTest.java)
   to run on emulators and devices.
 
+### Supported versions
+
+* Android SDK 30
+* Android SDK 31
+* Android SDK 32
+* Android SDK 33
+* Android SDK 34
+
+Android NDK is pinned to version 25.2.9519653 due to missing dependencies in later
+versions, please refer to this [cargo-ndk issue](https://github.com/bbqsrc/cargo-ndk/issues/94)
+for details. Also a thank you to the [meli project](https://github.com/p2panda/meli) for proposing
```

Comment:
So does it work for later NDK versions with this workaround, or not?

Reply:
the issue seems to be that newer versions of NDK do not link to `libgcc` anymore which breaks our app, so the old version of NDK is required during the build to explicitely link to its builtins library

the pinned version is also necessary to prevent cargo-ndk from using the most recent NDK version

I'm a bit fuzzy on the details though, but here is a more detailed discussion of the problem: https://github.com/mozilla/application-services/issues/5436

Reply:
that said as it's only a problem for a single given architecture that seems to be rare-ish (?) by all accounts we can also just drop the support for that architecture as an alternative